src/model/openthaigpt_pretraining_model/models/__init__.py

```diff
@@ -0,0 +1,31 @@
+from .constants import TOKENIZERS, MODELS, MODEL_CONFIGS
+
+
+def load_model(model_config):
+    tokenizer_object = TOKENIZERS.get(model_config.tokenizer.tokenizer_class, None)
+    if tokenizer_object is None:
+        raise NotImplementedError(
+            f"No tokenizer name: {model_config.tokenizer.tokenizer_class}"
+        )
+    model_config_object = MODEL_CONFIGS.get(model_config.name, None)
```

Comment:
Can load_model handle custom model not from huggingface ex. @MoosaTae  LLaMa with flash attention